components/count/package.json

```diff
@@ -22,10 +22,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```

Comment:
this is old - has been touched by many - "The govuk-frederic authors" would be more appropriate